tests/fixtures/testcases.json

```diff
@@ -0,0 +1,155 @@
+            "check": null,
+            "unique": false,
+            "references": null
+          },
+          {
+            "name": "sync_end",
+            "type": "timestamp",
+            "nullable": false,
+            "size": null,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          },
+          {
+            "name": "message",
+            "type": "varchar",
+            "nullable": false,
+            "size": 2000,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          }
+        ],
+        "table_name": "super_table",
+        "tablespace": null,
+        "schema": "prod",
+        "partitioned_by": [],
+        "alter": {},
+        "checks": [],
+        "index": [],
+        "primary_key": [
+          "data_sync_id",
+          "sync_start",
+          "sync_end",
+          "message"
+        ]
+      }
+    ]
+  },
+  {
+    "test_id": "caps_in_columns",
+    "ddl": "CREATE TABLE paths\n(\n    \"ID\"          int PRIMARY KEY,\n    \"TITLE\"       varchar   NOT NULL,\n    \"description\" varchar(160),\n    \"created_at\"  timestamp not null,\n    \"updated_at\"  timestamp\n);\n",
+    "parsed_ddl": [
+      {
+        "columns": [
+          {
+            "name": "\"ID\"",
+            "type": "int",
+            "nullable": false,
+            "size": null,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          },
+          {
+            "name": "\"TITLE\"",
+            "type": "varchar",
+            "nullable": false,
+            "size": null,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          },
+          {
+            "name": "\"description\"",
+            "type": "varchar",
+            "nullable": true,
+            "size": 160,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          },
+          {
+            "name": "\"created_at\"",
+            "type": "timestamp",
+            "nullable": false,
+            "size": null,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          },
+          {
+            "name": "\"updated_at\"",
+            "type": "timestamp",
+            "nullable": true,
+            "size": null,
+            "default": null,
+            "check": null,
+            "unique": false,
+            "references": null
+          }
+        ],
+        "primary_key": [
+          "\"ID\""
+        ],
+        "index": [],
+        "table_name": "paths",
+        "tablespace": null,
+        "schema": null,
+        "partitioned_by": [],
+        "alter": {},
+        "checks": []
+      }
+    ]
+  }
+]
```

Comment:
@xnuinside I have an idea of how to organize multiple tests in one file. Here is an example. The only thing I don't like about this approach is it's hard to read ddl query in JSON file.